examples/fluids/qfunctions/newtonian_types.h

```diff
@@ -50,6 +51,12 @@ typedef struct {
   CeedScalar temperature;
 } StatePrimitive;
 
+typedef struct {
+  CeedScalar chemical_potential;
+  CeedScalar velocity[3];
+  CeedScalar temperature;
```

Comment:
Do we not have more specific names for these?

Reply:
I'm not sure. Jed's suggestion in the past was to go the opposite route, and just name them `V1`, `V2`, `V3` or something like  that. I'm starting to think that might be best, since I don't know how to name them otherwise.

e.g. 
- The quantity I'm currently calling `temperature` is $-\frac{\rho}{P}$, which has units $\frac{kg}{J}$ or $\frac{s^2}{m^2}$. 
- The quantity I'm calling `velocity` is `\bm{U}\cdot\frac{\rho}{P}`, which has units $\bm{s}{m}$.

Reply:
I think the units for `V1` are $kg$ (since it has $\frac{\rho}{P}(e_{\text{kinetic}} + e_{\text{potential}})$)

Reply:
:shrug: Consistency is good so if we're going to name them, may as well make them match. I'm not sure it's easier in the code.

BTW, did you do anything here that may impact performance of the primitive or conservative variable code?

Reply:
Minorly, yes. The `StateFrom*` functions now also compute the entropy variables.

Reply:
Is that necessary? We added both conservative and primitive to avoid recomputing the equation of state each time that quantity is needed in fluxes, but I don't think entropy variables are used that way so we could probably remove the `V` member from `State`.

Reply:
How would we use entropy variables then? We use `StateFromQ` and `StateFromQ_fwd` all over the place, and that relies on the given state variables to be in `State`

Reply:
I don't think that I understand what you mean by "import entropy variables to that representation without making it part of `State`".

Tangentially, one of the major uses for entropy variables is in making the fluxes symmetric, which while not implemented here, would justify storing `V` in `State`.